p2panda-store/src/lib.rs

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+pub mod memory_store;
+pub mod traits;
+
+pub use memory_store::MemoryStore;
+pub use traits::{LogStore, OperationStore, StoreError};
+
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, PartialEq, Eq, std::hash::Hash, Deserialize, Serialize, Default)]
```

Comment:
Could you explain why you've included this `LogId` `struct`? Is it strictly necessary?

Reply:
It's actually an artefact left over from a previous design where the log id was required to be present in the operation extensions (`Extension<LogId>` was a required trait) but now I removed that requirement by adding `log_id` as an additional argument to `insert_operation()`.

TLDR; it isn't required, I deleted it :wink: